packages/browser-service/src/browser.service.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  CommonWalletConfig,
+  NetworkConfig,
+  WalletConfig,
+  WalletPage,
+  WalletTypes,
+} from '@lidofinance/wallets-testing-wallets';
+import {
+  Extension,
+  ExtensionService,
+} from '@lidofinance/wallets-testing-extensions';
+import { EthereumNodeService } from '@lidofinance/wallets-testing-nodes';
+import {
+  DEFAULT_BROWSER_CONTEXT_DIR_NAME,
+  WALLET_PAGES,
+} from './browser.constants';
+import {
+  BrowserContextService,
+  BrowserOptions,
+} from './browser.context.service';
+import { mnemonicToAccount } from 'viem/accounts';
+import { test } from '@playwright/test';
+
+type NodeConfig = {
+  rpcUrlToMock: string; // example: '**/api/rpc?chainId=1'
+};
+
+type BrowserServiceOptions = {
+  networkConfig: NetworkConfig;
+  walletConfig: WalletConfig;
+  nodeConfig: NodeConfig;
+  browserOptions?: BrowserOptions;
+  enableBrowserContext?: boolean;
```

Comment:
Why do we need this param if [`isCustomDirNeeded`](https://github.com/lidofinance/wallets-testing-modules/blob/4e6c020d5afc08efc9c7b83f68a1e7a24602d006/packages/browser-service/src/browser.service.ts#L113) fully depend on `!this.ethereumNodeService?.state`

So, for example if we pass `this.options.enableBrowserContext` as false it would create customDir even `this.ethereumNodeService?.state`=true

Reply:
if this.options.enableBrowserContext wil be false, and its will be not fork - we will use temporary dir)

Reply:
because we using `!this.ethereumNodeService?.state && this.options.enableBrowserContext;`
this.ethereumNodeService?.state = false and this.options.enableBrowserContext = false - will be created temp dir.
this.ethereumNodeService?.state = true and this.options.enableBrowserContext = false - will be created temp dir.
this.ethereumNodeService?.state = false and this.options.enableBrowserContext = true  - will be created custom dir.

Reply:
remove it, and make new customDirName with postfix by wallet name